pom.xml

```
@@ -422,6 +422,45 @@
                             </systemPropertyVariables>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>jpa</id>
+                        <phase>test</phase>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <dependenciesToScan>
+                                <dependency>jakarta.tck:persistence-platform-tck</dependency>
```

You just have the dependency artifact name incorrect. It should be:
```suggestion
                                <dependency>jakarta.tck:persistence-platform-tck-tests</dependency>
```